core/classes/Events/EventHandler.php

```diff
@@ -139,18 +202,24 @@ public static function executeEvent(string $event, array $params = []) {
     }
 
     /**
-     * Get a list of events.
+     * Get a list of events to display on the StaffCP webhooks page.
      *
+     * @param Language $language Language to translate event descriptions with.
      * @param bool $internal Whether to include internal events or not
      *
      * @return array List of all currently registered events.
      */
-    public static function getEvents(bool $internal = false): array {
+    public static function getEvents(Language $language, bool $internal = false): array {
         $return = [];
 
         foreach (self::$_events as $name => $meta) {
             if (!$meta['internal'] || $internal) {
-                $return[$name] = $meta['description'];
+                // The description might be an array if the event is a class based event
+                $description = is_array($meta['description'])
+                    ? $language->get(...$meta['description'])
```

Comment:
Event description need to be reworked, this does not support modules module languages

Example the forum events:
forum/new_topic_hook_info
forum/topic_reply